### PR TITLE
Adjust msg label confirmation modal to not check inactive children la…

### PR DESCRIPTION
**Pull request by norkans7**
…bels

Folder release handle the children labels properly
https://github.com/nyaruka/rapidpro/blob/master/temba/msgs/models.py#L1247

### CODE REVIEW

File: templates/msgs/label_delete.haml [hunk 1]
```diff
@@ -10,14 +10,9 @@
       .message
         {{error}}
 
-  -if object.children.all
```
Comment (norkans7) [inline templates/msgs/label_delete.haml:13]: children are turned `is_active=False`  so this is always TRUE

File: templates/msgs/label_delete.haml [hunk 2]
```diff
@@ -10,14 +10,9 @@
...
-    -blocktrans trimmed with name=object.name
-      <b>{{name}}</b> cannot be deleted since it is a parent to other labels.
+  -blocktrans trimmed with name=object.name
+    Are you sure you want to delete <b>{{name}}</b>? This cannot be undone.
```
Comment (rowanseymour) [inline templates/msgs/label_delete.haml:14]: Hmm should deleting a folder of labels delete the labels inside it? If you delete a group, we don't delete the contacts. Maybe this is different..but seems like we'd at least warn users here.

@nicpottier @ericnewcomer

Comment (nicpottier) [reply templates/msgs/label_delete.haml:14]: Feels like folders should only be deletable when the labels within them have been deleted to me.

Comment (rowanseymour) [reply templates/msgs/label_delete.haml:14]: Yeah I think this should explain to the user, if you want to delete this folder, first delete the labels

Comment (ericnewcomer) [reply templates/msgs/label_delete.haml:14]: I think I could be okay with deleting child labels if the confirmation showed them, but certainly forcing them to remove them one at a time is the safest bet.

Comment (norkans7) [reply templates/msgs/label_delete.haml]: I updated to only delete folder without active children labels

Comment (ericnewcomer) [reply templates/msgs/label_delete.haml]: Food for thought from our friends at gmail

Screen Shot 2020-12-09 at 10 14 20 AM

Comment (norkans7) [reply templates/msgs/label_delete.haml]: We fail when trying to delete a label used in flows, deleting on label at a time helps for to clean up the flows

**Comment by rowanseymour** [comment]
I think there was a little miscommunication here and @ericnewcomer was just adding the GMail example as a data point. We just want a clear error message that tells the user to delete the labels first in order to be able to delete the folder.

**Comment by ericnewcomer** [comment]
That's correct, I wasn't saying we should ape it, just adding what they do to the conversation.

**Pull request closed without merging**